cmd/renterd/main.go

```diff
@@ -126,10 +125,11 @@ var (
 		},
 		Autopilot: config.Autopilot{
 			Enabled:                        true,
-			RevisionSubmissionBuffer:       144,
 			AccountsRefillInterval:         defaultAccountRefillInterval,
 			Heartbeat:                      30 * time.Minute,
 			MigrationHealthCutoff:          0.75,
+			ContractConfirmationDeadline:   18,
```

Comment:
Making this configurable to users seems a bit odd to me. I think it's more confusing than anything.